v2/yarpcjson/inbound_test.go

```diff
@@ -127,15 +130,17 @@ func TestHandleSuccessWithResponseHeaders(t *testing.T) {
 		handler: reflect.ValueOf(h),
 	}
 
-	resw := new(yarpctest.FakeResponseWriter)
-	err := handler.Handle(context.Background(), &yarpc.Request{
+	reqBuf := &yarpc.Buffer{}
+	_, err := reqBuf.WriteString(`{"name": "foo", "attributes": {"bar": 42}}`)
+	require.NoError(err)
```

Comment:
nit: Not actionable now, but let's consider adding a test helper function to create these request buffers.

Reply:
I'd like to hold off on creating the helpers until the buffer is used in the HTTP transport, this way I can create several helpers in the `v2/yarpctest` package

Reply:
actually changed my mind and implemented these in #1566, 👍